.github/composite-actions/setup-base-version/action.yml

```diff
@@ -59,6 +59,28 @@ runs:
         sudo make USE_PGXS=1 PG_CONFIG=~/${{ inputs.install_dir }}/bin/pg_config install
       shell: bash
 
+    - name: Build PostGIS Extension
```

Comment:
Let's use a composite action as much as possible instead of copy-paste lots of linux commands.

Reply:
we have used Composite functions at most of the places, just places where others(eg: TDS_FDW) have used linux commands(because the previous version is checked out so it is not necessary that composite actions exist in previous version), only there we chose to proceed with linux commands